v20.1/authorization.md

```diff
@@ -28,6 +28,8 @@ By default, a new user belongs to the `public` role and has no privileges other
 ### `root` user
```

Comment:
I'm just including it here since this line doesn't show in the diff, but theres a block stating 
```
Use the CREATE USER and DROP USER statements to create and remove users, the ALTER USER statement to add or change a user's password, the GRANT <privileges> and REVOKE <privileges> statements to manage the user’s privileges, and the SHOW USERS statement to list users.
```

Can you update this to say `the ALTER USER statement to add or change a users password and role options`